concent_api/core/tests/test_unit_transfer_operation.py

```diff
@@ -109,27 +110,27 @@ def test_that_file_transfer_token_for_concent_is_never_out_of_date(self):
         )
         self.assertTrue(token.timestamp < token.token_expiration_deadline)
 
-    def test_that_download_file_transfer_token_for_golem_client_is_can_be_out_of_date(self):
+    def test_that_download_file_transfer_token_for_golem_client_cant_be_out_of_date(self):
         with freeze_time(self.time):
             new_time = self._get_deadline_exceeded_time_for_download_token(self.report_computed_task.size)
             with freeze_time(new_time):
-                download_token = create_file_transfer_token_for_golem_client(
-                    self.report_computed_task,
-                    self.authorized_client_public_key,
-                    FileTransferToken.Operation.download
-                )
-                self.assertTrue(download_token.timestamp > download_token.token_expiration_deadline)
-
-    def test_that_upload_file_transfer_token_for_golem_client_is_can_be_out_of_date(self):
+                with self.assertRaises(Http400):
+                    create_file_transfer_token_for_golem_client(
+                        self.report_computed_task,
+                        self.authorized_client_public_key,
+                        FileTransferToken.Operation.download
+                    )
+
+    def test_that_upload_file_transfer_token_for_golem_client_cant_be_out_of_date(self):
```

Comment:
this changes the bahaviuor (so it is no refactoring anymore) and must be clarified

Reply:
same as above.